aim/tests/base.py

```diff
@@ -54,7 +54,7 @@ def etcdir(*p):
 
 
 def sort_if_list(attr):
-    return sorted(attr) if isinstance(attr, list) else attr
+    pass
```

Comment:
If this API is not being used at all then we should just remove it.

Reply:
sure.